### DISCUSSION TITLE
Use EditorConfig for code styles

### DISCUSSION THREAD

**Discussion by myfreeer** [recp/cglm#285] [Ideas]
In [CONTRIBUTING.md](https://github.com/recp/cglm/blob/master/CONTRIBUTING.md) there are some guides for [Code Style](https://github.com/recp/cglm/blob/master/CONTRIBUTING.md#code-style), [EditorConfig](https://editorconfig.org/) can make it easier for developers to configure their editors to code styles required per project.

**Comment by recp** [comment]
Hi @myfreeer, I didn't use EditorConfig before but seems cool, yes we could have it ;)
